opossum-tool/OpossumUI
# Add original source field to manual attributions

### Summary of changes

The original source name is kept when promoting a signal to an attribution.

### Context and reason for change

We would like to see original source for suggested (=MERGER) signals

### How can the changes be tested

See updated unit & e2e test.
Take a file with an origin source, and promote it to an attribution. Check that the source is saved in the file, and that the file with the new fields can be loaded.
